tests/NSubstitute.Analyzers.Tests.CSharp/DiagnosticAnalyzerTests/NonSubstitutableMemberReceivedAnalyzerTests/ReceivedAsExtensionMethodTests.cs

```diff
@@ -668,4 +668,28 @@ public void Test()
 
         await VerifyNoDiagnostic(source);
     }
+
+    public override async Task ReportsNoDiagnostics_WhenSubscribingToEvent(string method)
+    {
+        var source = $@"using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+using System;
+namespace MyNamespace
+{{
+    public class Foo
+    {{
+        public event Action SomeEvent;
```

Comment:
As per [this comment](https://github.com/nsubstitute/NSubstitute.Analyzers/issues/202#issuecomment-1445237062) I think this should report [NS1001](https://github.com/nsubstitute/NSubstitute.Analyzers/blob/master/documentation/rules/NS1001.md).